core/tests/protractor_utils/general.js

```diff
@@ -128,16 +134,25 @@ var moveToEditor = async function() {
 };
 
 var expect404Error = async function() {
-  expect(await element(by.css('.protractor-test-error-container')).getText()).
+  var protractorTestErrorContainer =
```

Comment:
```suggestion
  var errorContainer =
```

`protractor-test-*`  is a common pattern for all protractor classes so you don't need to mention that explicitly in the variables.

Reply:
Ditto below.

Reply:
Done